libcore/src/KeyboardLayout.h

```diff
@@ -74,7 +74,7 @@ class CORE_LIB_EXPORT KeyboardLayout : public QObject
 		};
 		Q_ENUM(Finger)
 
-		enum Row
+		enum class Row
```

Comment:
I'd like to avoid using `enum class` because QML doesn't support it.

In QML, the enum can be accessed like this:
```qml
KeyboardLayout.Row_A
```
assuming that `KeyboardLayout` is an **instance** of the `KeyboardLayout` C++ class (because enums won't work without an instance).

I can't access the enum like this from QML:
```qml
KeyboardLayout.Row.Row_A
```

Using `enum class` changes it only for C++.